### DISCUSSION TITLE
Rustdesk recent nightly releases block "turn off the display"

### DISCUSSION THREAD

**Discussion by mikekgr** [rustdesk/rustdesk#6615] [General]
### Bug Description

Dear Sirs,
I have discovered a nasty bug let me explain. I am running Rustdesk (nightly releases) in several Windows 11 laptops. In all of these laptops I have set up Power Plan Settings Turn off the display in 5 minutes and this is working fine until one connection to under testing laptop, e.g. I connected (using Rustdesk) to this laptop for little time and I disconnected then. After this action the laptop screen is remaining on, does not following the Power Plan Settings. If I stop Rustdek service (by exiting Rustdesk) then the normal operation of the Power Plan Settings is restored. Even if I start Rustdek and select to run it as service, the correct Power Plan Settings is preserved. Now if I get an external Rustdesk connection - disconnection to this laptop then the screen never goes off.
Please solve this problem, it is important to me, Thanks

### How to Reproduce

See above

### Expected Behavior

To follow Power Plan Settings.

### Operating system(s) on local side and remote side

Windows 11

### RustDesk Version(s) on local side and remote side

Version: 1.2.4   Build date: 2023-12-03 00:26

### Screenshots

not necessary

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/pull/6520

**Reply by mikekgr** [reply]
Correct me if I am wrong but as far I understand #6520 is related to keep screen on during connection. My problem is AFTER the disconnection, then it is never follows Power Plan Settings in my case "Turn off the display in 5 minutes" never works except I exit Rustdesk and I reload again

**Reply by rustdesk** [reply]
@21pages verify this

**Reply by rustdesk** [reply]
@mikekgr please try out tomorrow's nightly build, https://github.com/rustdesk/rustdesk/pull/6623

**Reply by mikekgr** [reply]
Thanks a lot for the update. I will test it tomorrow and I will let you know.
Keep your excellent work !

**Reply by mikekgr** [reply]
Dear Sirs, I am glad to confirm that the bug has been solved.
Thanks for your great work.
